core/src/main/java/feast/core/config/CoreLoggingConfig.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright 2018-2020 The Feast Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package feast.core.config;
+
+import feast.common.logging.AuditLogEntry.FeastInstance;
+import feast.common.logging.AuditLogEntry.FeastInstance.FeastComponent;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.info.BuildProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ComponentScan("feast.common")
```

Comment:
Which components are being picked up here?

Reply:
The `AuditLogger` and `GrpcMessageInterceptor`. https://github.com/feast-dev/feast/pull/891/files#diff-e5ba13ce49d6f28bcc91d30359f8daa4R35

Reply:
Ok. Just wondering if we should be a bit more specific in how we structure the common lib. So that we can have something like `feast.common.config.logging` as the scan target, but I guess this is fine for now.

Reply:
Made `feast.common.logging` as a scan target.